elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -886,6 +886,8 @@ elytron.filesystem-realm.encoded=Whether the identity names should be stored enc
 elytron.filesystem-realm.identity=An identity which can be managed by a security realm.
 elytron.filesystem-realm.hash-encoding=The string format for the password if it is not stored in plain text.
 elytron.filesystem-realm.hash-charset=The character set to use when converting the password string to a byte array.
+elytron.filesystem-realm.credential-store=The reference to the credential store that contains the secret key to encrypt and decrypt the realm.
```

Comment:
I suggest to update the description of `elytron.filesystem-realm.encoded` above - when encryption is set the `encoded` attribute is ignored (confirmed in the analysis).

Reply:
Updated